### PR TITLE
dockerized insightface

### PR DESCRIPTION
**Pull request by armanrahman22**

### CODE REVIEW

File: algorithms/insightface/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.6
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+COPY vectorize.py .
+COPY insightface.pb .
```
Comment (c-w) [inline algorithms/insightface/Dockerfile]: This file seems to not be part of the pull request.

**Review by c-w — CHANGES_REQUESTED** [review]
Please also add the new algorithm to the makefile in the [build-algorithms](https://github.com/c-w/faceanalysis/blob/c4ad866adc0976ea819caa90694ed2a24efdd0ce/Makefile#L15) and [release-algorithms](https://github.com/c-w/faceanalysis/blob/c4ad866adc0976ea819caa90694ed2a24efdd0ce/Makefile#L24) recipes.

**Comment by armanrahman22** [comment]
added insight face to  build-algorithms and release-algorithms.

**Review by c-w — APPROVED** [review]
